azure-quantum/azure/quantum/qiskit/job.py

```diff
@@ -212,15 +212,20 @@ def _format_ionq_results(self, sampler_seed=None):
 
     @staticmethod
     def _qir_to_qiskit_bitstring(obj):
-        """Convert the data structure from Azure into the "schema" used by Qiskit """
+        """Convert the data structure from Azure into the "schema" used by Qiskit"""
         if isinstance(obj, str) and not re.match(r"[\d\s]+$", obj):
             obj = ast.literal_eval(obj)
 
         if isinstance(obj, tuple):
             # the outermost implied container is a tuple, and each item is
             # associated with a classical register. Azure and Qiskit order the
             # registers in opposite directions, so reverse here to match.

```

Comment:
nit: as far as I understood, the registers are no longer in opposite direction, so this comment is no longer valid?